src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsEmbeddedFractures.cpp

```diff
@@ -489,9 +489,7 @@ void SinglePhasePoromechanicsEmbeddedFractures::updateState( DomainPartition & d
 
       arrayView1d< real64 const > const area = subRegion.getElementArea().toViewConst();
 
-      arrayView2d< real64 > const & fractureTraction = subRegion.template getField< fields::contact::traction >();
-
-      arrayView1d< real64 >  const & dTdpf = subRegion.template getField< fields::contact::dTraction_dPressure >();
+      arrayView2d< real64 > const & fractureContactTraction = subRegion.template getField< fields::contact::traction >();
```

Comment:
I gave a different name called fractureContactTraction.  if you prefer to effectiveTraction, i can change it. @CusiniM

Reply:
I think I prefer `effectiveTraction`. I even think that we should change the name of the field that gets written in teh output.  That would actually be more important coz this is just an internal name

Reply:
i agree. we should refactor it. e.g., two distinct traction fields, one is total while the other is effective.  It will make the meaning much clear.